js/damage.js

```diff
@@ -120,9 +120,13 @@ function getDamageResult(attacker, defender, move, field) {
     if (typeEffectiveness === 0) {
         return {"damage":[0], "description":buildDescription(description)};
     }
-    if (move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200))) {
+    if (move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200) || field.isGravity)) {
         return {"damage":[0], "description":buildDescription(description)};
     }
+    if (move.name === "Synchronoise" &&
```

Comment:
the sky drop change was only added because I noticed I missed it when making the Synchronoise addition, I didn't think of adding more as I thought they would have already been covered.